lib/Graph/NodeValue.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "glow/Graph/NodeValue.h"
+#include "glow/Graph/Graph.h"
```

Comment:
Why do we need to add the graph.h here?  Can you wrap the logic in a cpp function and use forward decleration method to avoid the extra include?

Reply:
Well, there's a statement getNode()->getParent()->getLogContext() that will actually dereference the Function class, which makes forward declaration unable to work here